SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -683,3 +669,58 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+bool UGlobalStateManager::ProcessOpListForServersCanBeginPlay(const TArray<Worker_OpList*>& OpLists)
+{
+	const Worker_AddComponentOp* AddOp = nullptr;
+	const Worker_AuthorityChangeOp* AuthOp = nullptr;
+
+	for (Worker_OpList* OpList : OpLists)
+	{
+		for (size_t i = 0; i < OpList->op_count; ++i)
+		{
+			const Worker_Op* Op = &OpList->ops[i];
+
+			if (Op->op_type == WORKER_OP_TYPE_ADD_COMPONENT &&
+				Op->add_component.data.component_id == SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID)
+			{
+				AddOp = &Op->add_component;
+			}
+			else if (Op->op_type == WORKER_OP_TYPE_AUTHORITY_CHANGE &&
+				Op->authority_change.component_id == SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID)
+			{
+				AuthOp = &Op->authority_change;
+			}
+		}
+	}
+
+	if (AddOp == nullptr && AuthOp == nullptr)
+	{
+		return false;
+	}
+
+	// verify assumption that we should never receive only the auth op here
+	check(AddOp || AuthOp == nullptr);
+
+	ApplyStartupActorManagerData(AddOp->data);
+	
+	// Capture the value of CanBeginPlay we received over the wire, because the AuthorityChanged() call below will change it
+	const bool bOriginalCanBeginPlay = bCanBeginPlay;
+
+	if (AuthOp)
+	{
+		check(AuthOp->authority == WORKER_AUTHORITY_AUTHORITATIVE);
+
```

Comment:
Explicitly handling these ops here as a special case is better than trying to add new functionality into the Dispatcher for this one case.